packages/next/src/build/webpack/loaders/lightningcss-loader/src/minify.ts

```diff
@@ -0,0 +1,135 @@
+// @ts-ignore
+import { ModuleFilenameHelpers } from 'next/dist/compiled/webpack/webpack'
+import { webpack } from 'next/dist/compiled/webpack/webpack'
+import { RawSource, SourceMapSource } from 'webpack-sources'
```

Comment:
Import from pre-compiled version of `webpack-sources`: `next/dist/compiled/webpack-sources1` or `next/dist/compiled/webpack-sources3`